static/app/components/themeAndStyleProvider.tsx

```diff
@@ -61,6 +62,12 @@ export function ThemeAndStyleProvider({children}: Props) {
 
   const theme = config.theme === 'dark' ? darkTheme : lightTheme;
 
+  const didPrintBanner = useRef(false);
+  if (!didPrintBanner.current && NODE_ENV !== 'development') {
+    didPrintBanner.current = true;
+    printConsoleBanner(theme.colors.blue400, theme.text.familyMono);
```

Comment:
Future-proofing the theme access

```suggestion
    printConsoleBanner(theme.tokens.content.accent, theme.font.family.mono);
```